### DISCUSSION TITLE
an error in join to git hub education

### DISCUSSION THREAD

**Discussion by farokyosry** [community/community#111378] [GitHub Education]
### Hi everyone,

I tried more than once to enter the university’s photo, but it would take me back to the page again and not write anything to me.

**Comment by ettaboyle** [comment]
Hi @farokyosry,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. This post violates our Community Code of Conduct, including **posting the same comment** in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

Please update[ your other post](https://github.com/orgs/community/discussions/111377) with any additional details, instead of opening a new discussion. If you continue to post duplicate content we will block your account for 7 days and repeat violations of the code of conduct will result in a permanent account ban from the Community Discussions.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
